consensus/double_sign.go

```diff
@@ -70,6 +70,14 @@ func (consensus *Consensus) checkDoubleSign(recvMsg *FBFTMessage) bool {
 
 						now := big.NewInt(time.Now().UnixNano())
 
+						leaderShardKey := *shard.FromLibBLSPublicKeyUnsafe(consensus.LeaderPubKey)
+						leaderAddr, err := subComm.AddressForBLSKey(leaderShardKey)
+						if err != nil {
+							consensus.getLogger().Err(err).Str("msg", recvMsg.String()).
```

Comment:
how often this case been happening, how possible to check with custom node against ostn ?

Reply:
I don't have the double-sign setup to test this. can you?